packages/flutter_tools/test/protocol_discovery_test.dart

```diff
@@ -88,13 +91,16 @@ void main() {
         logReader.addLine('Observatory not listening...');
         final Uri timeoutUri = Uri.parse('http://timeout');
         final Uri actualUri = await uriFuture.timeout(
-            const Duration(milliseconds: 100), onTimeout: () => timeoutUri);
+            const Duration(milliseconds: 100),
+            onTimeout: () => timeoutUri);
```

Comment:
nit: probably more readable if you put a comma after `timeoutUri` and then the `)` on the next line.

in general, i'm guessing you ran `dartfmt`. That's fine, but you may need to do some hand-tweaking after to avoid some of the less readable output it produces. Generally I find it works best with a very wide line wrapping option.